test/data/ansible/check_ostree.yaml

```
@@ -275,7 +275,7 @@
 
     # case: check running container with podman
     - name: run ubi8 image
-      command: podman run ubi8-minimal:latest cat /etc/redhat-release
+      command: podman run registry.access.redhat.com/ubi8/ubi8-minimal:latest cat /etc/redhat-release
```

```suggestion
      command: podman run registry.access.redhat.com/ubi8/ubi-minimal:latest cat /etc/redhat-release
```

This is the official address according to https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?gti-tabs=unauthenticated&container-tabs=gti